core/templates/dev/head/css/oppia.css

```diff
@@ -2646,6 +2646,36 @@ md-card.preview-conversation-skin-supplemental-card {
   margin: 0;
   padding: 0;
 }
+/* Editor Tutorial Welcome Modal */
+.oppia-welcome-modal .modal-dialog {
+  position: fixed;
+  margin: 0;
+  width: 100%;
+  height: 100%;
+  padding: 0;
+}
+
+.oppia-welcome-modal .modal-content {
+  border-radius: 0;
+  height: 100%;
+}
+
+.oppia-welcome-modal .modal-body,
+.oppia-welcome-modal .oppia-long-text p {
+  text-align: center;
+}
+
+.oppia-welcome-modal .modal-body {
+  width: 70%;
+  top: 20%;
+  margin: 0 auto;
+}
+
+.divider {
```

Comment:
Also, this divider thing feels a tad odd. Have you considered alternatives like floating the rightmost button to the right within a container, or applying a margin to one of the buttons?